task_12/js/class.DateX.js

```diff
@@ -0,0 +1,66 @@
+class DateX extends Date {
```

Comment:
You will have problem with transpiling this code to ES5. It will work only browser will have native ES6 support.
Inheritance from native classes isn't supported by Babel.